lib/builder.js

```diff
@@ -409,6 +409,25 @@ Builder.prototype.bundle = function(expressionOrTree, outFile, opts) {
 
   var outputOpts = processOutputOpts(opts, { outFile: outFile });
 
+  // override the fetch function if given
+  if (opts) {
+    var loaderFetch = self.loader.fetch;
+    self.fetch = opts.fetch;
+    self.loader.fetch = function (load) {
+      var args = Array.prototype.slice.call(arguments, 0);
+      args.push(loaderFetch.bind(self.loader));
+
+      var source = (self.fetch || loaderFetch).apply(self.loader, args);
```

Comment:
I think it's fine to use `(self.fetch || loaderFetch).call(self.loader, load, loaderFetch)` which cuts out the above two lines.